### ISSUE TITLE
Support for _react= selector strategy

### ISSUE THREAD

**Issue by dreamercz** [MarketSquare/robotframework-browser#1519]
Hi, I was browsing through Playwright docs and noticed that React selectors are experimentally supported. https://playwright.dev/python/docs/selectors/#react-selectors Is there a plan to add this selector strategy to Browser as well?

**Describe the solution you'd like**
Would be nice to have this added to the available selector strategies https://marketsquare.github.io/robotframework-browser/Browser.html#Finding%20elements so something like `Keyword    _react=Tab[labe="Home"]` would be possible.

**Describe alternatives you've considered**
I guess I could write a JS extension that accomplishes the same by circumventing Browser and calling Playwright directly.

Thank you for this project by the way. I used to work with Selenium for a long time and it is great to have such a powerful library that goes a different way!

**Comment by aaltat** [comment]
We pass in the selector as is (unless it's frame piercing selector) to the playwright side. Because we don't alter the selector, I would assume I would assume that everything listed in NodeJS would just work out of the box.

**Comment by dreamercz** [comment]
Interesting, the selector strategy indeed works, I should've realized that. And I managed to assert visibility over some of the react components, but not for others. I tried it with just "Playwright for Python" and it's the same problem.

What I am trying to do exactly: 
1) I Downloaded and built (unminified) React app from https://github.com/alexanmtz/material-sense
2) With RF Browser I'm trying to do the following:
```
*** Settings ***
Library   Browser

*** Variables ***
${page}    http://localhost:8000

*** Test Cases ***

Check React elements
    Setup browser and context
    New Page    ${page}
    Wait For Elements State
    ...    _react=Tab[label = "Home"]
```
I do something similar with Playwright for Python with the same results. In both cases, the assertion times out because for some reason the element is deemed not visible.

However, for some elements of the app it works, for example asserting visibility of `_react=Topbar` works just fine.

I realize now that this is not connected to the RF Browser library but rather it might be something in Playwright proper. React selectors are after all an experimental feature.
